kite-web-element/src/components/auth/AuthForm.vue

```diff
@@ -2,6 +2,7 @@
   <el-form
     :model="authForm"
     :rules="authRules"
+    @submit.native.prevent="submitForm('authForm')"
```

Comment:
Is this supposed to be here? I thought this prevents you from submitting the form if there is only one input element, which doesn't apply to the authForm?

Reply:
No it doesn't prevent you from submitting the form, but it _also_ shouldn't be there because it does nothing.